cmd/eval.go

```diff
@@ -69,6 +69,41 @@ func newEvalCommandParams() evalCommandParams {
 	}
 }
 
+func validateEvalParams(p *evalCommandParams, cmdArgs []string) error {
```

Comment:
NOTE: Now I see why the struct embedding is being done. I wonder if we should factor these shared params into their own struct?

Reply:
So you're thinking like a `evalCommandParams`, `benchCommandParams`, and `sharedEvalCommandParams` and then to validate we do each separately?

Reply:
Yeah, I was thinking to use composition over inheritance but like my comment above mentions, it's not required.